coldfront/core/project/forms.py

```diff
@@ -18,6 +18,17 @@
     'EMAIL_DIRECTOR_EMAIL_ADDRESS', '')
 
 
+
+class ProjectCreateForm(forms.ModelForm):
+    class Meta:
+        model = Project
+        fields = ['title', 'pi', 'description', 'field_of_science']
+        
+    pi = PrincipalInvestigatorField(
```

Comment:
Although `pi` is a well-known term, `principal_investigator` is more accessible and easier to maintain than `pi.`

This is just a recommendation.

Reply:
Good call out

Reply:
Actually, that name is bound by the name of the field in the build in coldfront model, so not something we can really change